src/identity-providers/add/DetailSettings.tsx

```diff
@@ -228,6 +276,51 @@ export const DetailSettings = () => {
                 </FormAccess>
               </ScrollForm>
             </Tab>
+            <Tab
+              id="mappers"
+              eventKey="mappers"
+              title={<TabTitleText>{t("common:mappers")}</TabTitleText>}
+            >
+              <KeycloakDataTable
+                emptyState={
+                  <ListEmptyState
+                    message={t("identity-providers:noMappers")}
+                    instructions={t("identity-providers:noMappersInstructions")}
+                    primaryActionText={t("identity-providers:addMapper")}
+                    //   onPrimaryAction={() => setAddDialogOpen(true)}
+                  />
+                }
+                loader={loader as any}
```

Comment:
There should be no need to cast this.
```suggestion
                loader={loader}
```